app/questionnaire/path_finder.py

```diff
@@ -154,17 +140,11 @@ def get_location_path(self, group_id=None, group_instance=0):
             group_id = SchemaHelper.get_first_group_id(self.survey_json)
 
         routing_path = self.get_routing_path(group_id, group_instance)
-        can_reach_summary = self.can_reach_summary(routing_path)
-
-        location_path = [Location(group_id, 0, block_id) for block_id in self.preceeding_path]
-
-        location_path += routing_path
 
-        if can_reach_summary:
-            for block_id in PathFinder.CLOSING_INTERSTITIAL_PATH:
-                location_path.append(Location(SchemaHelper.get_last_group_id(self.survey_json), 0, block_id))
+        if self.can_reach_summary(routing_path):
+            routing_path.append(Location(SchemaHelper.get_last_group_id(self.survey_json), 0, 'thank-you'))
 
-        return location_path
+        return routing_path
```

Comment:
The function is called get_location_path and is returning routing path, one needs to be changed